### DISCUSSION TITLE
What happens when your Herd Pro license expires ?

### DISCUSSION THREAD

**Discussion by vesper8** [beyondcode/herd-community#478] [Q&A] [answered]
I couldn't find this information on the website so I'm asking here. If you get a Herd Pro license for 1 year, what happens when it expires? Do you get to continue using the software but are locked out of any new updates? Or do you lose access to the Pro features as soon as the license expires?

**Comment by turiondolby** [comment]
> You are purchasing a license for one year and you can renew the license yearly or it automatically switches back to the free version.

You can see this if you go to purchase Herd Pro https://herd.laravel.com/checkout

![image](https://github.com/beyondcode/herd-community/assets/8764796/4b1646e1-5c12-4998-a5b6-ba8033f0f61c)

**Comment by eqxDev** [comment]
Do you lose access to your databases then...

**Comment by nousename** [comment]
and no update pricing for the next year?
always new price

**Comment by sschlein** [comment, marked as answer]
I've added this to the FAQ for the updated Herd website that we release with the new features that were teased at Laracon US. 

Herd downgrades to the free version if you don't renew. We'll send you notifications to make sure that this doesn't happen and you can renew in time or have enough time to move your databases. To be honest: If you are only using Herd Pro because it ships with a database, I highly recommend just using the free version and running a database on the side. There's no need to pay us yearly for MySQL.

**Comment by Skintillion** [comment]
The Herd page no longer says it only lasts a year. There is no mention of the expiry anywhere.

**Reply by sschlein** [reply]
Hey, it's on the pricing table and also mentioned in the checkout. "A license for one year" and in the checkout text as well. If you're on auto renew, it's also next to the buy button. 

We also send multiple reminders via email so that there's definitely not an accidental charge after a year and provide refunds in case you miss them. 

No worries, we don't want money from users who aren't happy 😊
